lib/dom.js

```diff
@@ -1114,6 +1115,16 @@ declare class HTMLLabelElement extends HTMLElement {
     control: HTMLElement | null;
 }
 
+declare class HTMLLinkElement extends HTMLElement {
+    crossorigin: 'anonymous' | 'use-credentials';
```

Comment:
sorry, one more thing: I think the DOM API is `crossOrigin` even though the attribute name is `crossorigin`. see the DOM interface section in https://html.spec.whatwg.org/multipage/semantics.html#the-link-element